app/controllers/users.ts

```diff
@@ -1,10 +1,11 @@
 import { NextFunction, Request, Response } from 'express';
+import HttpStatus from 'http-status-codes';
+
 import userService from '../services/users';
 import { User } from '../models/user';
-import { statusCodes } from './commons';
 import { notFound } from '../errors';
 
-export function getUsers(_: Request, res: Response, next: NextFunction): Promise<Response | void> {
+export function getUsers(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
```

Comment:
If `req` is unused, `_` is not forced by linter ?

Reply:
no, it isn't. Besides, im not a big fan of using underscore for unused vars in js. For example, if you have more than one you have to name the other one.

Reply:
Yeah more than one is awful, it was only a question because the linter of JS does that